tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -683,9 +687,9 @@ def ratio_value_number_to_time_series_length(x):
 @not_apply_to_raw_numbers
 def fft_coefficient(x, c, param):
     """
-    Calculates the fourier coefficients of the one-dimensional discrete Fourier Transform for real input by fast
```

Comment:
Just for clarity. (Smaller lines are usually more readable). Changed it back to original